### DISCUSSION TITLE
ExplorerPatcher seems to work now in Windows 11 Update 24H2.

### DISCUSSION THREAD

**Discussion by mogulman52** [valinet/ExplorerPatcher#4010] [General]
When I initially installed Windows Update 24H2 it complained about ExplorerPatcher.  I uninstalled it and tried again and it failed again for a different reason.  I reinstalled ExplorerPatcher. I ran the Windows Update Troubleshooter.  Select Start > Settings  > System  > Troubleshoot  > Other troubleshooters. Next, under Most frequent, select Windows Update > Run.  I left the PC running overnight with ExplorerPatcher installed.  It automatically ran the Windows Update 24H2 overnight and everything is working fine this morning including ExplorerPatcher.

**Comment by pyrates999** [comment]
If you could post the _different reason_ that it failed, that be helpful I think.

**Reply by mogulman52** [reply]
The other reason was

Windows 11 24H2 Updates Not Working - Install Error  - 0x80070057

**Reply by pyrates999** [reply]
this could be the reason why: https://answers.microsoft.com/en-us/windows/forum/all/windows-10-update-error-0x80070057/7d6490a6-790e-4aab-9b89-9b436f88167a

**Reply by paulhypno** [reply]
Windows 11, version 24H2

https://support.microsoft.com/en-gb/windows/inside-this-update-93c5c27c-f96e-43c2-a08e-5812d92f220d?OCID=WUSettings

Has failed twice (0xc1900209) and shows reason as Explorer Patcher stopping install

**Comment by paulhypno** [comment]
Mine too didn't like that Explorer Patcher was installed and wouldn't update Windows. So I ran the troubleshooter that you suggested but still Windows Update won't get past 14% before giving me the same notification as before.
I'm tempted to uninstall EP, update my Windows 24H2, then install EP again.
Has anyone done this without problems?

**Reply by pyrates999** [reply]
what update are you trying to install?

**Reply by Ruskie-it** [reply]
He said. Windows 24H2.
And I have the same problem

**Reply by pyrates999** [reply]
"update my windows 24h2" implies he already upgraded to windows 11 24h2.  If it's the windows 11 24h2 upgrade, then yes you must uninstall EP, then you can upgrade to windows 11 24h2.  Then install EP again and setup its settings again.

If it's a normal windows update, then you don't have to uninstall EP first.

**Comment by paulhypno** [comment]
Thanks I'll have to do that. Yes I have already upgraded to Windows 11 (as in the title of the thread) Strange thing is that I haven't met this on my Desktop PC, just my Laptop. Maybe an update to EP further down the line will resolve this in future Windows updates.

**Reply by pyrates999** [reply]
Always update to the latest version of EP.  Old versions may not work on newer versions of windows due to the integration with windows that EP does.

**Reply by paulhypno** [reply]
Thank you for your prompt replies. Yes, I was using the latest EP version. 
So, this is what I've done:
I uninstalled EP and the Windows 11 update correctly installed. But after the update I've found that I no longer have the annoyances brought in by Windows 11 (eg drag and drop; separation of browser windows and Explorer windows on the taskbar etc) All the things that I appreciated from EP I now seem to have without it. 
It would appear that Microsoft has now allowed these possibilities?
Anyway I'll run my laptop without the brilliant EP and see if there's anything I'm missing by not having it.

**Comment by jlegg4** [comment]
Uninstalled EP, successfully updated to 24H2 and reinstalled EP.  The Windows 10 (ExplorerPatcher) Toolbar seems to run perfectly.  No problem installing the Windows patch on Tuesday, 12/10.  So far, so good.

**Reply by TimH32** [reply]
I don't see a difference between the Windows 11 taskbar and Windows 10 (ExplorerPatcher) taskbar.  When I select just "Windows 10" I get a message that "Windows 10 taskbar is not available in this version of Windows".

The only reason I've been using EP is to get back support for the quick launch toolbar.  I don't see the toolbar option with "Windows 10 (ExplorerPatcher)" taskbar selected.

Does anyone know if the toolbar support will return via EP or is it now gone for good?

**Reply by pyrates999** [reply]
check this: https://github.com/valinet/ExplorerPatcher/discussions/3774#discussion-7289141

**Reply by TCHarp01** [reply]
How did you manage to get the EP installer to run?  My box threw out an error when I attempted to re-install EP.
![Untitled](https://github.com/user-attachments/assets/0fe942e4-6e39-442b-adca-7fa813375f45)

**Reply by pyrates999** [reply]
Uninstall EP first before upgrading to 24h2. Then install the latest EP from here: https://github.com/valinet/ExplorerPatcher/releases

**Comment by Bortsamsonhk** [comment]
I have uninstalled the latest WIN11 updates, uninstalled EP then reinstalled both to no avail. I just want my task bar back on the side.... :(

**Reply by pyrates999** [reply]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

please see this for more info: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

**Reply by Bortsamsonhk** [reply]
thank you so much! really appreciate it.

**Reply by coyote3** [reply]
> Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**. We no longer offer more support on the stock Windows 10 taskbar.

I switched to the **Windows 10 (ExplorerPatcher)** version, but I still can't install 24H2.  Perhaps I could if I uninstalled EP, but I'm not eager to do that since it sounds like I then wouldn't be able to reinstall EP on 24H2.  Can this situation please get fixed?  I'd like to get onto 24H2.

**Reply by pyrates999** [reply]
Yes, you must uninstall EP first before upgrading to 24h2.  Then install the latest EP from here: https://github.com/valinet/ExplorerPatcher/releases

**Reply by coyote3** [reply]
Thank you very much, pyrates999.  I decided to not heed the reports of not being able to re-install EP, so I uninstalled EP then upgraded to 24h2 then installed EP and all was well after I followed [these instructions to reinstall the Quick Launch Toolbar](https://github.com/valinet/ExplorerPatcher/discussions/3774#discussion-7289141).

Thank goodness we don't have to settle for Microsoft's terrible taskbar!

**Comment by rostek2** [comment]
Windows 11, version 23H2
Installtion error - 0xc1900209

I've tried to use tricks with add exceptions to the defender through the windows power shell, but it didn't solve the problem.

So for today (2024.12.15) there is not possible to go to the 24H2 with installed EP?

**Reply by pyrates999** [reply]
You're not supposed to have EP installed before you upgrade to 24h2.  Uninstall EP, then upgrade to 24h2.  Then install EP again from here: https://github.com/valinet/ExplorerPatcher/releases

**Reply by rostek2** [reply]
But the question was:
- for today (2024.12.15) there is not possible to go to the 24H2 with installed EP?

**Reply by pyrates999** [reply]
No.

**Reply by rostek2** [reply]
Does anybody know if settings export from EP in Win11 23H2 and then import into EP in Win11 24H2 will work?
BTW - do I need to add exceptions (with use of Windows Power Shell) to the Windows defender?

**Reply by pyrates999** [reply]
I would make a note of all the settings changed in EP properties application instead.

And yes, you do need to run those powershell commands to add exclusions to windows defender.  If you are using a third party antivirus program, you must add those exclusions to the antivirus program instead.

**Reply by rostek2** [reply]
Too late to note all settings... I'm during Win11 24H2 installation :-)

Thank you for your quick responses. Hopefully EP will work with Win11 24H2 and Classic Shell .

**Reply by rostek2** [reply]
Unfortunatelly, after Win11 24H2 installation - Explorer Patcher is not working.

It was able to install, but not working. It means that instead od Explorer Patcher taskbar view - I see Win11 settings of the taskbar.

Import settings is not working.

**Reply by rostek2** [reply]
![obraz](https://github.com/user-attachments/assets/7cf70a1b-fa25-4826-bd76-e64ce8b2cfa5)

**Reply by pyrates999** [reply]
Open up EP Properties application, go to taskbar, and set **taskbar style** to **Windows 10 (ExplorerPatcher)**, _not_ **Windows 10**.  We no longer offer more support on the stock Windows 10 taskbar.

please see this for more info: https://github.com/valinet/ExplorerPatcher/wiki/ExplorerPatcher's-taskbar-implementation

Uninstall classic shell. That is no longer maintained.

The replacement is open shell.

**Comment by rostek2** [comment]
Another disadvantage is:
- Quick Launch toolbar disappeared (but you can restore it manually)
https://www.tenforums.com/tutorials/4624-add-remove-quick-launch-toolbar-windows-10-a.html


Advantages:
- you can switch Taskbar style to: Windows 10 (Explorer Patcher)  and it works

So finally after Win11 24H2 installation, the Explorer Patcher is working fine.

P.S
I've noticed that "Windows 11" logo disapeared from the shutdown window - did you notice that also?

**Reply by Amrsatrio** [reply]
Yes. And apparently they've been experimenting on backporting this to Windows 10 22H2.

**Comment by FormicolaCE** [comment]
So, if I understand it .... if you export your EP settings (through EP), then unistall EP, then upgrade tp 24h2. You can then install the latest version of EP , import your previous settings from the saved file and you back to where want to be ?

**Reply by TimH32** [reply]
Correct... except that after you install latest EP and then import your previous settings you'll need to make one properties change... change the Taskbar style from "Windows 10" to "Windows 10 (ExplorerPatcher)", unless you want the Windows 11 taskbar.

Good luck!
